lib/npm-check-updates.js

```diff
@@ -193,19 +188,17 @@ function analyzeProjectDependencies(options, pkgData, pkgFile) {
     print(options, `Fetching ${vm.getVersionTarget(options)} versions...`, 'verbose');
 
     return vm.upgradePackageDefinitions(current, options).spread((upgraded, latest) => {
-
         const newPkgData = vm.upgradePackageData(pkgData, current, upgraded, latest, options);
 
         // print
         if (options.json) {
-
             // use the upgraded dependencies data to generate new package data
             // INVARIANT: we don't need try-catch here because pkgData has already been parsed as valid JSON, and vm.upgradePackageData simply does a find-and-replace on that
-            var output = options.jsonAll ? jph.parse(newPkgData) :
+            const output = options.jsonAll ? jph.parse(newPkgData) :
```

Comment:
@kibertoad This breaks the code since `output` is returned from `analyzeProjectDependencies`. Was this a work in progress? Let me know if you are still working on a PR and I will avoid commenting. When a PR is ready, make sure to run the tests first and ensure they all pass! Thanks!

Reply:
I am not sure why the [travis output](https://travis-ci.org/tjunnone/npm-check-updates/builds/482049203) didn't show up in the PR!